modules/home/assets/js/components/home-screen.js

```diff
@@ -19,12 +19,12 @@ const HomeScreen = ( props ) => {
 				/>
 				<Box sx={ { display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, justifyContent: 'space-between', gap: 3 } }>
 					<Stack sx={ { flex: 1, gap: 3 } }>
-						<GetStarted getStartedData={ props.homeScreenData.get_started } />
+						<GetStarted getStartedData={ props.homeScreenData.get_started[ 0 ] } />
```

Comment:
We'd prefer, if we could keep the old data structure inside the localized data.

Reply:
get started will be taken care of in a separate task